src/corecel/Macros.hh

```diff
@@ -42,6 +42,33 @@
 #    define CELER_FORCEINLINE inline
 #endif
 
+#define CELER_COMP_UNKNOWN 0
+#define CELER_COMP_CLANG 1
+#define CELER_COMP_GCC 2
+#define CELER_COMP_INTEL 3
+#define CELER_COMP_MSVC 4
+#define CELER_COMP_NVCPP 5
```

Comment:
If you're adding these, perhaps it would be good to update the rest of the macros to use them?

Reply:
it's a bit verbose since you have to compare two macros. In a `#if` directive I think it's more ergonomic to check for the builtin compiler macro directly